src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkComparables.tsx

```diff
@@ -0,0 +1,79 @@
+import { Column, Flex, Join, Message, Spacer, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MetaTags } from "Components/MetaTags"
+import { ArtistAuctionResultItemFragmentContainer } from "Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem"
+import { extractNodes } from "Utils/extractNodes"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import { MyCollectionArtworkComparables_artwork } from "__generated__/MyCollectionArtworkComparables_artwork.graphql"
+interface MyCollectionArtworkComparablesProps {
+  artwork: MyCollectionArtworkComparables_artwork
+}
+
+const ComparablesContainer: React.FC<MyCollectionArtworkComparablesProps> = ({
+  artwork,
+}) => {
+  if (!artwork.auctionResult) {
+    return <Flex />
```

Comment:
**(suggestion):** Not sure if this has any consequences, but we usually return `null` in case the component is not supposed to render because of missing data.

```suggestion
    return null
```

Reply:
Hmm, the logic further down the component suggests that we display the message `There aren’t any auction results available by the artist at this time.` if there are no comparables available. Is this early return really needed?

Reply:
Agree. Done